### PR TITLE
Completed ticket #13 - responds with a list of endpoints when accessi…

### PR DESCRIPTION
**Pull request by ActuallySilver**
…ng /api endpoint. Not all endpoints are in the list but are currently being added

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -22,7 +22,12 @@ describe("/api", () => {
   });
   describe("GET", () => {
     test("Returns a 200 with no body", () => {
-      return request(app).get("/api").expect(200);
+      return request(app)
+        .get("/api")
+        .expect(200)
+        .then(({ body }) => {
+          expect(body).toEqual(expect.objectContaining({ endpoints: expect.any(Object) }));
```
Comment (Poonam-raj) [inline __tests__/app.test.js]: would expect a more explicit expect line here, you know the endpoints you expect to see here but this test checks next to nothing

File: app/controllers/api.controller.js [hunk 2]
```diff
@@ -1,3 +1,31 @@
...
+        exampleResponse: {
+          articles: [
+            {
+              title: "Seafood substitutions are increasing",
+              topic: "cooking",
+              author: "weegembump",
+              body: "Text from the article..",
+              created_at: 1527695953341,
+            },
+          ],
```
Comment (Poonam-raj) [inline app/controllers/api.controller.js:26]: keep updating (as you've mentioned)

**Review by Poonam-raj — CHANGES_REQUESTED** [review]
Please make that test a bit more specific

**Review by Angaca — APPROVED** [review]
Looks good!

**Pull request merged**